modules/web/src/app/wizard/component.ts

```diff
@@ -350,7 +355,12 @@ export class WizardComponent implements OnInit, OnDestroy {
         template.nodeDeployment.name = namePrefix;
         this._clusterSpecService.initializeClusterFromClusterTemplate(template);
         this._nodeDataService.initializeNodeDataFromMachineDeployment(template.nodeDeployment);
-        this.applicationService.applications = template.applications;
+        template.applications.forEach(app => {
+          if (app.spec.applicationRef.name === CLUSTER_AUTOSCALING_APP_DEF_NAME) {
+            (app.labels ??= {})[ApplicationLabel.ManagedBy] = ApplicationLabelValue.KKP;
+          }
```

Comment:
Why do we need this? Isn't `labels` already part of ApplicationInstallation object?

Reply:
in cluster templates, applications are saved inside an annotation named: `kubermatic.io/initial-application-installations-request:` and it doesn't have the labels in the object

Reply:
So it was broken previously?
Also, speaking of cluster templates, did you test if the updated cluster autoscaler values are persisted when editing/customizing cluster template?